src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -873,7 +873,8 @@ void emitWarningMessages(EnergyPlusData &state,
     }
 }
 
-void ShowFatalError(EnergyPlusData &state, std::string const &ErrorMessage, OptionalOutputFileRef OutUnit1, OptionalOutputFileRef OutUnit2)
+[[noreturn]] void
+ShowFatalError(EnergyPlusData &state, std::string const &ErrorMessage, OptionalOutputFileRef OutUnit1, OptionalOutputFileRef OutUnit2)
```

Comment:
Marking it `[[noreturn]]` is really nice because that tells compilers that once you call this function, it's over.  And that way you don't have to have workarounds of unused code after calls to ShowFatalError.